fix: (sfui2-900) chip docs [ hotfix] (#2531)
vuestorefront/storefront-ui: apps/docs/components/components/chip.md

---
# Chip

> ::: warning
> This is a component preview. Whilst this component is a part of NPM package, it's still considered experimental and its API might hugely change based on user feedback.
> :::
> 

Reviewer: ```suggestion
::: warning This is an experimental component
This component is shipped in our NPM package, but its API might change based on user feedback.
:::
```

![image](https://user-images.githubusercontent.com/18535681/228077134-1d1c03ec-6b38-45b1-81ee-a7bf1fd08817.png)

Author: thx :)
